views/user/view.php

```diff
@@ -43,20 +46,27 @@
                         ]) ?>
                     </p>
                 <?php endif ?>
-            </div>
-            <?php if (count($authClients) > 0): ?>
-            <div class="col-sm-4">
-                <div class="well well-sm">
-                    <h2>Connect extra profiles:</h2>
-                    <?= yii\authclient\widgets\AuthChoice::widget([
-                        'baseAuthUrl' => ['site/auth'],
-                        'popupMode' => false,
-                        'clients' => $authClients,
-                    ]) ?>
+
+                <!-- <?php if (count($authClients) > 0): ?>
```

Comment:
Why is it commented out?

Reply:
Because it is removed from new design:
https://github.com/samdark/yiifeed-redesign/blob/master/screens/desktop/admin/user.png

I can display it, just tell me where would you put it.

Reply:
Ah... right. It's OK as is.